### DISCUSSION TITLE
zoxide + fzf: different behavior

### DISCUSSION THREAD

**Discussion by ralph-bergmann** [ajeetdsouza/zoxide#717] [Q&A] [answered]
On MacOS, I installed zoxide and fzf using homebrew. This is the part of my `.zshrc`:
```
[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
eval "$(zoxide init zsh --cmd cd)"
```

I'm facing some strange behaviour, I can't imagine where it comes from and how to fix it :-(

This is what zoxied has learned so far:
```
zoxide query -l -s
  40.0 /Users/ralphbergmann/development/projects/flutter/set...
  16.0 /Users/ralphbergmann/development
  10.0 /Users/ralphbergmann/development/projects/flutter
  10.0 /Users/ralphbergmann/development/projects/dart/inj...
   8.0 /Users/ralphbergmann/development/projects
   4.0 /Users/ralphbergmann/development/projects/android
   4.0 /Users/ralphbergmann/development/projects/ios/ber...
   2.0 /Users/ralphbergmann/development/projects/android/ber...
   2.0 /Users/ralphbergmann/development/projects/ios
   2.0 /Users/ralphbergmann/development/projects/dart
   2.0 /Users/ralphbergmann/.config/nvim
   0.5 /Users/ralphbergmann/.config
```

When I run `cdi flutter` it opens the fzf window where I can choose the final path. But when I run `cdi projects` it jumps directly to that folder. Why this different behaviour?

**Comment by ajeetdsouza** [comment, marked as answer]
That's a shortcut - by default, `cdi` will jump to a destination instead of bringing up fzf if there is only one possible match.
